tests/globalhelper/helper.go

```diff
@@ -277,3 +278,24 @@ func AllowAuthenticatedUsersRunPrivilegedContainers() error {
 
 	return nil
 }
+
+// CopyFiles copy file from source to destination.
+func CopyFiles(src string, dst string) error {
+	originalFile, err := os.Open(src)
+	if err != nil {
+		return err
+	}
+
+	defer originalFile.Close()
+
+	newFile, err := os.Create(dst)
+	if err != nil {
+		return err
```

Comment:
Same here: `fmt.Errorf("failed to create file %s: %w", dst, err)`

Reply:
thanks